manifests/config.pp

```diff
@@ -78,11 +52,9 @@
     }
   }
 
-  file { '/etc/default/marathon':
-    ensure  => 'present',
-    content => template('marathon/default.erb'),
-    owner   => $owner,
-    group   => $group,
-    mode    => '0644',
+  case $::osfamily {
+    'Debian': { include ::marathon::config::sysvinit }
```

Comment:
We need to contain or anchor these classes to be able to rely on their ordering relative to the other install steps.
Something like this should do:
```puppet
'Debian': {
  include marathon::config::sysvinit
  contain 'marathon::config::sysvinit'
}
'Redhat': {
  include marathon::config:: systemd
  contain 'marathon::config:: systemd'
}
```
It's a pain.. but this is puppet :(

Also, just nitpicking.. but I'd prefer we stay consistent and don't use a leading `::` in front of class names. Maybe we should be, but in that case I'd rather make another PR and do it everywhere.